app/config/test_checker.py

```diff
@@ -0,0 +1,21 @@
+from django.test import Client
```

Comment:
Two concerns about the file:
1. Shouldn't it be in `tests` folder like all the others?
2. Should it be `test_urls` to reflect that it checks stuff in `urls.py`? On the other hand, we also have `test_attribution_model.y` even though it then should be `test_models.py`. So fine with both, just wanted to raise the point because I am used to have one test file per module (like `mymodule.py` --> `test_mymodule.py`)

Reply:
I intentionally didn't add a folder as I don't expect to have more tests in here, since this is the `config` folder after all.

Reply:
I moved it to a folder. Why not. Keep the same pattern across all apps :+1: